source/utilities.tex

```diff
@@ -6904,44 +6746,17 @@
   template<class T, class U>
     bool operator==(const shared_ptr<T>& a, const shared_ptr<U>& b) noexcept;
   template<class T, class U>
-    bool operator!=(const shared_ptr<T>& a, const shared_ptr<U>& b) noexcept;
-  template<class T, class U>
-    bool operator<(const shared_ptr<T>& a, const shared_ptr<U>& b) noexcept;
-  template<class T, class U>
-    bool operator>(const shared_ptr<T>& a, const shared_ptr<U>& b) noexcept;
-  template<class T, class U>
-    bool operator<=(const shared_ptr<T>& a, const shared_ptr<U>& b) noexcept;
-  template<class T, class U>
-    bool operator>=(const shared_ptr<T>& a, const shared_ptr<U>& b) noexcept;
+    strong_ordering operator<=>(const shared_ptr<T>& a, const shared_ptr<U>& b) noexcept;
 
   template<class T>
     bool operator==(const shared_ptr<T>& x, nullptr_t) noexcept;
-  template<class T>
-    bool operator==(nullptr_t, const shared_ptr<T>& y) noexcept;
-  template<class T>
-    bool operator!=(const shared_ptr<T>& x, nullptr_t) noexcept;
-  template<class T>
-    bool operator!=(nullptr_t, const shared_ptr<T>& y) noexcept;
-  template<class T>
-    bool operator<(const shared_ptr<T>& x, nullptr_t) noexcept;
-  template<class T>
-    bool operator<(nullptr_t, const shared_ptr<T>& y) noexcept;
-  template<class T>
-    bool operator>(const shared_ptr<T>& x, nullptr_t) noexcept;
-  template<class T>
-    bool operator>(nullptr_t, const shared_ptr<T>& y) noexcept;
-  template<class T>
-    bool operator<=(const shared_ptr<T>& x, nullptr_t) noexcept;
-  template<class T>
-    bool operator<=(nullptr_t, const shared_ptr<T>& y) noexcept;
-  template<class T>
-    bool operator>=(const shared_ptr<T>& x, nullptr_t) noexcept;
-  template<class T>
-    bool operator>=(nullptr_t, const shared_ptr<T>& y) noexcept;
+
 
   // \ref{util.smartptr.shared.spec}, \tcode{shared_ptr} specialized algorithms
   template<class T>
     void swap(shared_ptr<T>& a, shared_ptr<T>& b) noexcept;
+  template<class T>
+    strong_ordering operator<=>(const shared_ptr<T>& x, nullptr_t) noexcept;
```

Comment:
Should have been added to the redundant empty line above.

Reply:
Fixed.